libs/rest/rest-common/src/main/java/net/corda/httprpc/messageBus/package-info.java

```diff
@@ -0,0 +1,4 @@
+@Export
+package net.corda.httprpc.messageBus;
```

Comment:
Should this be 
```suggestion
package net.corda.httprpc.messagebus;
```
[Java naming conventions](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) for packages say _"Package names are written in all lower case to avoid conflict with the names of classes or interfaces."_